R/serialization.R

```diff
@@ -10,7 +8,33 @@ Serialize <- function(model, filename) {
   model_serialization_function <-
     switch(attributes(model)$type,
       "GaussianKernel" = SerializeGaussianKernelPtr,
```

Comment:
Technically, the `GaussianKernel` is only needed for the test binding.  It doesn't hurt to leave it here, but if you aren't planning on including the test binding in the release, it might be nice to remove the model type from here too (hopefully automatically).  Up to you. :+1: